src/window/dxgi/src/lib.rs

```diff
@@ -19,20 +19,19 @@ extern crate log;
 extern crate kernel32;
```

Comment:
please remove both `kernel32` and `user32` dependencies from here as no longer used